problemsets-mat223/modules/module13-exercises.tex

```diff
@@ -1,17 +1,33 @@
 \begin{exercises}
+%Patrick:
+    %45min
 	\begin{problist}
-		\prob  Example problem with parts
-		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
+		\prob Let $\mathcal A=\Set{\vec a_1,\vec a_2}$ where $\vec a_1=\mat{2\\1}_{\mathcal E}$ and $\vec a_2=\mat{1\\-2}_{\mathcal E}$ and let $\mathcal B=\Set{\vec b_1,\vec b_2}$ where $\vec b_1=\mat{3\\-1}_{\mathcal E}$ and $\vec b_2=\mat{-2\\3}_{\mathcal E}$ be bases for $\R^2$. 
+	    \begin{enumerate}
+	        \item Given that $[\vec x]_{\mathcal A}=\mat{1\\-1}$, find $[\vec x]_{\mathcal B}$.
+	        \item Find the change of basis matrix $M_{A}^{B}$.
```

Comment:
There's a command for change of basis matrix. Look through the module for it. (Also, don't invent notation in the problems! This is what you did in MAT224, but it's not what we do in MAT223.)